src/packagedcode/npm.py

```diff
@@ -578,6 +578,10 @@ def _parse(cls, json_data, package_only=False):
             if value:
                 extra_data[extra_data_field] = value
 
+        overrides = json_data.get('overrides')
+        if overrides:
+            extra_data['overrides'] = overrides    
```

Comment:
This is okay as a start but we need to do a bit more here.
See the go replace directives PR and reviews which is a similar issue: https://github.com/aboutcode-org/scancode-toolkit/pull/3693#discussion_r1527842940